spec/classes/foreman_repo_spec.rb

```diff
@@ -46,7 +50,8 @@
         let :params do
           {
             repo: '1.19',
-            configure_scl_repo: false
+            configure_scl_repo: false,
+            yum_repo_baseurl: 'https://baz.fi'
```

Comment:
I'm a big fan of using `example.com` for these cases. Also, trailing comma's are great
```suggestion
            yum_repo_baseurl: 'https://example.com',
```

Reply:
Sure, I'll patch the other cases, too.

Reply:
It's fixed (sorry, my sub-conscience typed `example.org` instead of `example.com`; both are valid IANA-approved example domains, though). 

In one case I'm using `examplekeys.org` to verify that the code is actually taking the `keypath` literally and not mixing it with `baseurl`.

Reply:
My point was indeed that an IANA example domain should be used.